api/src/main/java/marquez/db/DatasetDao.java

```diff
@@ -42,9 +43,14 @@
 @RegisterRowMapper(DatasetMapper.class)
 public interface DatasetDao extends BaseDao {
   @SqlQuery(
-      "SELECT EXISTS ("
+      "WITH symlink AS (\n"
```

Comment:
_Minor_: `symlink` -> `symlinks`

Reply:
code got removed when switching to `dataset_view`